mycore-mets/src/main/java/org/mycore/mets/resource/MCRAltoHighlightResource.java

```diff
@@ -187,11 +197,13 @@ private JsonObject buildPositionObject(String altoWord) {
         JsonObject positionObject = new JsonObject();
         String plainWord = altoWord.replaceAll("<em>|</em>", "");
         String[] split = plainWord.split("\\|");
-        positionObject.addProperty("content", split[0]);
-        positionObject.addProperty("xpos", Integer.parseInt(split[1]));
-        positionObject.addProperty("vpos", Integer.parseInt(split[2]));
-        positionObject.addProperty("width", Integer.parseInt(split[3]));
-        positionObject.addProperty("height", Integer.parseInt(split[4]));
+        if(split.length == 5){
```

Comment:
Is this "if" still necessary if we filter empty word before?
But it "does not hurt", if we keep it this way.

Reply:
I put the if for some robustness, for reasons where the incomming string can be corrupt. Maybe not nessescary.